extensions/ql-vscode/src/local-databases-ui.ts

```diff
@@ -410,6 +411,23 @@ export class DatabaseUI extends DisposableObject {
     );
   };
 
+  private handleTourDependencies = async (): Promise<void> => {
+    // find workspace root folder
+    if (!workspace.workspaceFolders?.length) {
+      throw new Error("No workspace folder is open.");
+    } else {
+      // find tutorial queries folder
+      const tutorialQueriesPath = Uri.parse(
+        `${workspace.workspaceFolders?.[0].uri}/tutorial-queries`,
+      );
```

Comment:
Any reason why you are (re-)parsing this URI to add a path to it? Would this be better?

```suggestion
      const tutorialQueriesPath = path.join(workspace.workspaceFolders[0].uri, "tutorial-queries");
```

Also, no need for `?.` since you checked above that there is at least one element.

Reply:
You're absolutely right, I went a bit deep on debugging https://github.com/github/vscode-codeql/pull/2104 so I wanted to be extra sure file paths weren't the issue when I was seeing a large list of errors. When I was working on introducing skeleton packs, I had some trouble getting these paths to work on Windows/Linux/MacOS. 

I've switched back to using a `join` 👍  Thanks!